packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.scss

```diff
@@ -2,31 +2,99 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-@import '~@itwin/itwinui-css/scss/variables';
+@import "~@itwin/itwinui-css/scss/variables";
 
 .gmw-group-property-action-container {
   display: flex;
   flex-direction: column;
   overflow-x: auto;
   height: 100%;
+  gap: $iui-baseline;
 
   .gmw-property-options {
     display: inherit;
     flex-direction: inherit;
     gap: $iui-baseline;
-    margin-bottom: $iui-baseline * 4;
   }
+}
 
-  .gmw-property-selection-container {
+.gmw-properties-list {
+  display: flex;
+  flex-direction: column;
+  gap: 7.5px;
```

Comment:
There should be a variable that will match what you need.

Reply:
Not for vertical spacing, $iui-baseline is typically used. All the other spacing variables are meant to be used in horizontal spacing.